openapi/models/openapi_access.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo import models, fields, api
+
+
+class Access(models.Model):
+    _name = 'openapi.access'
+    _description = 'Access via API '
+
+    active = fields.Boolean('Active', default=True)
+    namespace_id = fields.Many2one('openapi.namespace', 'Integration', required=True)
+    model_id = fields.Many2one('ir.model', 'Model', required=True)
+    model = fields.Char(related='model_id.model')
+    api_create = fields.Boolean('Create via API', default=False)
+    api_read = fields.Boolean('Read via API', default=False)
+    api_update = fields.Boolean('Update via API', default=False)
+    api_delete = fields.Boolean('Delete via API', default=False)
+    read_one_id = fields.Many2one(
+        'ir.exports',
+        'Read One Fields',
+        help='Fields to return on reading one record',
+        domain="[('resource', '=', model)]")
+    read_many_id = fields.Many2one(
+        'ir.exports',
+        'Read Many Fields',
+        help='Fields to return on reading via non one-record endpoint',
+        domain="[('resource', '=', model)]")
+    create_context_ids = fields.Many2many(
+        'openapi.access.create.context',
+        string='Creation Context Presets',
+        help="Can be used to pass default values or custom context",
+        domain="[('model_id', '=', model_id)]",
+        context="{'default_model_id': model_id}",
+    )
+
+    _sql_constraints = [
+        ('namespace_model_uniq',
+         'unique (namespace_id, model_id)',
+         'There is already a record for this Model')
```

Comment:
"You already have defined this model on this integration."

Reply:
There might be more than one administrators :smiley: